chaincode/ledger.go

```diff
@@ -179,6 +180,15 @@ type Testtuple struct {
 	Tag           string      `json:"tag"`
 }
 
+type ComputePlan struct {
+	AggregatetupleKeys      []string  `json:"aggregatetupleKeys"`
+	AssetType               AssetType `json:"assetType"`
+	CompositeTraintupleKeys []string  `json:"compositeTraintupleKeys"`
+	Status                  string    `json:"status"`
+	TesttupleKeys           []string  `json:"testtupleKeys"`
+	TraintupleKeys          []string  `json:"traintupleKeys"`
```

Comment:
I think we will need a `tags` or `labels` field for a compute_plan

Reply:
I also think it would be great to support multiple tags/labels.
We could do an another PR doing the change for all the existing assets having a `tag` field?

Reply:
I agree it should be `tags` and not `tag` but I also agree it should be on a different PR. Even more so since it will break compatibility with the backend.

And for the `tags` property on compute plan, I will add it after successfully reproduce the current compute plan capability. Good idea !

Thanks for the feedbacks